### DISCUSSION TITLE
Create ui.text

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1364] [Q&A]
i have this text "Tipo de negociação: TESTE		 Em litígio judicial: Não" and put it in ui.text component (through content parameter)
what i seen on the screen is "Tipo de negociação: TESTE Em litígio judicial: Não" it is removing the spaces in the midle, how can solve this?

**Comment by mturoci** [comment]
Thanks for the question @Rmfjd1978. Please use [discussions](https://github.com/h2oai/wave/discussions) for questions and [issues](https://github.com/h2oai/wave/issues) for feature requests / bug reports in future.

 > it is removing the spaces in the middle

This is the default HTML behavior, you can see that the same happened even in your Github comment. If you really need it though, you can wrap your text in `pre` tags.

```py
q.page['example'] = ui.form_card(
    box='1 1 4 10',
    items=[ui.text('<pre>aa        aa</pre>')]
)
```

**Comment by Rmfjd1978** [comment]
is it possible to allign a ui.text on the right?

**Reply by mturoci** [reply]
ui.text (as a container) can be aligned, but if you mean the textual contents, that's currently possible only via small HTML hacks.
